view/desktop/src/components/PageLoader.tsx

```diff
@@ -0,0 +1,96 @@
+import { useEffect, useState } from "react";
+
+const tips = [
+  "The statusbar color can be changed in the appearance settings",
+  "You can change the order of widget actions. Try it!",
+  "Lorem ipsum dolor sit amat. Met the statusbar color can be changed in the appearance settings",
+];
+
+export const PageLoader = () => {
+  const [tip, setTip] = useState(tips[0]);
+  useEffect(() => {
+    const interval = setInterval(() => {
+      const randomTip = tips[Math.floor(Math.random() * tips.length)];
+      setTip(randomTip);
+    }, 5000);
+
+    return () => clearInterval(interval);
+  }, []);
+
+  return (
+    <div className="relative flex h-screen w-screen flex-col items-center justify-between bg-white pt-4">
+      <div className="fixed top-0 h-36 w-screen" data-tauri-drag-region />
+
+      <div className="flex h-full flex-col items-center justify-center gap-5">
+        <div className="animate-[spin_1s_ease-in-out_infinite] ">
+          <svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" fill="none" viewBox="0 0 32 32">
```

Comment:
Let’s separate the SVG preloader into a shared UI component since we’ll be using it in other places